app/views/user_emailer/send_welcome_email.html.erb

```diff
@@ -0,0 +1,214 @@
+      font-weight: 200;
+    }
+    h1 {
+      font-size: 36px;
+    }
+    h2 {
+      font-size: 28px;
+    }
+    h3 {
+      font-size: 22px;
+    }
+    p, ul, ol {
+      margin-bottom: 10px;
+      font-weight: normal;
+      font-size: 14px;
+    }
+    ul li, ol li {
+      margin-left: 5px;
+      list-style-position: inside;
+    }
+    /* ---------------------------------------------------
+    RESPONSIVENESS
+    Nuke it from orbit. It's the only way to be sure.
+    ------------------------------------------------------ */
+    /* Set a max-width, and make it display as block so it will automatically stretch to that width, but will also shrink down on a phone or something */
+    .container {
+      display: block!important;
+      max-width: 600px!important;
+      margin: 0 auto!important; /* makes it centered */
+      clear: both!important;
+    }
+    /* Set the padding on the td rather than the div for Outlook compatibility */
+    .body-wrap .container {
+      padding: 20px;
+    }
+    /* This should also be a block element, so that it will fill 100% of the .container */
+    .content {
+      max-width: 600px;
+      margin: 0 auto;
+      display: block;
+    }
+    /* Let's make sure tables in the content area are 100% wide */
+    .content table {
+      width: 100%;
+    }
+  </style>
+</head>
+
+<body bgcolor="#f6f6f6">
+
+  <!-- body -->
+  <table class="body-wrap">
+    <tr>
+      <td></td>
+      <td class="container" bgcolor="#FFFFFF">
+
+        <!-- content -->
+        <div class="content">
+          <table>
+            <tr>
+              <td>
+                <h1>Thanks for signing up for TechHire, <%= @user.company.organization %>!</h1>
+                <p>Someone from Opportunity At Work will contact you soon with more details.
+                  In the meantime, you can login with these account details to get your
+                  company profile up and running.</p>
+                <table>
+                  <tr>
+                    <td class="padding">
+                      <p>Username:  <%= @user.email %></p>
+                      <p>Password: <%= @password %></p>
+                    </td>
+                  </tr>
+                </table>
+              </td>
+            </tr>
+          </table>
+        </div>
+        <!-- /content -->
+
+      </td>
+      <td></td>
+    </tr>
+  </table>
+  <!-- /body -->
+
+  <!-- footer -->
+  <table class="footer-wrap">
+    <tr>
+      <td></td>
+      <td class="container">
+
+        <!-- content -->
+        <div class="content">
+          <table>
+            <tr>
+              <td align="center">
+                <p>Don't like these annoying emails? <a href="#"><unsubscribe>Unsubscribe</unsubscribe></a>.
+                </p>
+              </td>
+            </tr>
+          </table>
+        </div>
+        <!-- /content -->
+
+      </td>
+      <td></td>
+    </tr>
+  </table>
+  <!-- /footer -->
+
+</body>
+</html>
```

Comment:
This came from someone's github, who has already tested this basic email out on several email clients. We might want to add/change some styling, but this will look pretty nice to start out. I'll tack in a screenshot as a comment...